### PR TITLE
Added functionality to calculate min_height when the datatable is under 10 items

### PR DESCRIPTION
**Pull request by ucg8j**
Per [issue 25](https://github.com/plotly/dash-table-experiments/issues/25) this PR fixes the sizing.

### CODE REVIEW

File: src/components/DataTable.react.js [hunk 1]
```diff
@@ -210,6 +210,13 @@ class DataTable extends Component {
         return this.state.rows;
     }
 
+    getMinHeight() {
+        if (this.getSize() < 10) {
+            return (this.getSize() * 35) + 35 + 15; // 35px extra edded for the filter box and 15px added for padding
+        }
+        return (typeof DataTable.defaultProps.min_height != 'undefined') ? DataTable.defaultProps.min_height : 350;
```
Comment (chriddyp) [inline src/components/DataTable.react.js]: `!=` => `!==`

**Comment by chriddyp** [comment]
This looks good to me @ucg8j ! Just one small comment. Also, could you bump the minor version number to `0.5.1` (https://github.com/plotly/dash-table-experiments/blob/master/dash_table_experiments/version.py) and add an entry in the `CHANGELOG.md` file: https://github.com/plotly/dash-table-experiments/blob/master/CHANGELOG.md?

**Comment by ucg8j** [comment]
Hi @chriddyp - all done. Let me know if there's anything else. Big thanks to @RenzoDS too!

**Comment by chriddyp** [comment]
Great, thanks @ucg8j ! Just one last comment here: https://github.com/plotly/dash-table-experiments/pull/27#discussion_r156166599

**Comment by ucg8j** [comment]
Done :)

**Comment by chriddyp** [comment]
Thanks @ucg8j !

**Pull request merged**